README.md

```diff
@@ -9,6 +9,50 @@ Travis. You will need to be a member of
 [kubernetes/test-infra-maintainers](https://github.com/orgs/kubernetes/teams/test-infra-maintainers)
 to merge.
 
+## Changing jobs
```

Comment:
Hummm, This probably need to be under #Testing Kubernetes section. I would not know what `Changing Jobs` means, since this is the repo README.